source/gloperate-assimp/source/AssimpSceneLoader.cpp

```diff
@@ -178,9 +187,22 @@ PolygonalGeometry * AssimpSceneLoader::convertGeometry(const aiMesh * mesh) cons
         geometry->setNormals(std::move(normals));
     }
 
+	if (mesh->HasTextureCoords(0))
+	{
+		// Copy texture cooridinate array
+		std::vector<glm::vec3> textureCoordinates;
+		for (size_t i = 0; i < mesh->mNumVertices; ++i)
+		{
+			const auto & textureCoordinate = mesh->mTextureCoords[0][i];
+			textureCoordinates.push_back({ textureCoordinate.x, textureCoordinate.y, textureCoordinate.z });
```

Comment:
I found this issue for ```map``` and ```set``` but not for ```vector```. Here is a blog entry claiming that it works with gcc 4.7: http://lemire.me/blog/archives/2012/06/20/do-not-waste-time-with-stl-vectors/